### PR TITLE
Add red wine facts

### PR DESCRIPTION
**Pull request by 597abhinav**

### CODE REVIEW

File: facts.txt [hunk 1]
```diff
@@ -41,4 +41,8 @@
         "Chicken is a rich source of two B vitamins: niacin or vitamin B3 and vitamin B6. A 4-ounce (1 ounce = 28.3495g) portion meets 40 and 16 percent respectively of the recommended daily value, or DV.",
         "Like beef, fish and other types of poultry, chicken is high in protein. Each ounce of meat provides 7g protein."
     },
-}
+    "Red Wine": {
+	"While beer makes that unaesthetic beer belly, wine does not affect your waistline at all.",
+	"Red wines are known to contain many beneficial antioxidants such as polyphenol and resveratrol that have cardio-protective effects and anti-cancer properties. Grape skin is especially rich in antioxidants. "
+    }
```
Comment (rahuldecoded) [inline facts.txt]: Comma is missing.

**Comment by rahuldecoded** [comment]
Update your repo

**Comment by rahuldecoded** [comment]
Update your repo.

**Comment by 597abhinav** [comment]
by updating my repo what do you mean?
should i work on master branch only and not the one I created?

**Comment by rahuldecoded** [comment]
There is merge conflict I'm getting

**Pull request closed without merging**